### PR TITLE
Added: the add box button on the edit packaging modal(#1x0eg0t)

### PR DESCRIPTION
**Pull request by ymaheshwari1**
**Screenshot**

image

### CODE REVIEW

**Comment by azkyakhan** [comment]
LGTM sir

File: src/views/EditPackagingModal.vue [hunk 1]
```diff
@@ -65,7 +65,13 @@
     </ion-card> 
 
     <ion-list>
-      <ion-list-header>{{ $t("Boxes") }}</ion-list-header>
+      <ion-item lines="none">
```
Comment (azkyakhan) [inline src/views/EditPackagingModal.vue:68]: LGTM sir

**Review by azkyakhan — APPROVED** [review]

File: src/views/EditPackagingModal.vue [hunk 2]
```diff
@@ -65,7 +65,13 @@
...
+        <ion-label>{{ $t('Boxes') }}</ion-label>
```
Comment (dt2patel) [inline src/views/EditPackagingModal.vue:69]: we should use ion-note here

**Review by dt2patel — CHANGES_REQUESTED** [review]
requested one change in tag.

File: src/views/EditPackagingModal.vue (hunk 2 above)
Comment (ymaheshwari1) [reply src/views/EditPackagingModal.vue]: Sir, replaced ion-label with ion-note.

**Review by dt2patel — APPROVED** [review]
looks good

**Pull request merged**